install-deps.sh

```diff
@@ -11,35 +11,34 @@ function debs() {
     if [ -n "${USE_CLANG}" ]; then
         extra=clang
     fi
-    if [ "$ID" = "ubuntu" ]; then
-        if [ ! -f /usr/bin/add-apt-repository ]; then
-            apt-get -y install software-properties-common
-        fi
-        add-apt-repository -y ppa:ubuntu-toolchain-r/test
-    fi
+
```

Comment:
we should remove dead code. This was required for gcc6 and above at some point. but now seastar needs gcc7 maybe? i have to double check.

Reply:
install-deps tries for gcc-8. it looks like gcc7 might work with seastar, but i'm seeing a lot of recent posts people are using gcc-8. we can revert back depending on what the conclusion is.